modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/IgniteSchema.java

```diff
@@ -44,7 +44,7 @@ public class IgniteSchema extends AbstractSchema {
     /** Constructor. */
     public IgniteSchema(String name, int version, Collection<? extends IgniteDataSource> tables) {
         this.name = name;
-        this.version = version;
+        this.catalogVersion = version;
```

Comment:
Perhaps it makes sense to rename the parameter too?
(and the SCHEMA_VERSION constant in the ExecutionServiceImplTest test)

Reply:
done